5.6/action.yml

```diff
@@ -0,0 +1,13 @@
+# action.yml
+name: 'Prestashop PHP 5.6 Linter'
+description: 'PHP linter for 5.6'
+inputs:
+  folder-to-exclude:
+    description: 'Folder to exclude'
+    required: false
+    default: '! -path "./vendor/*"'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  args:
+    - ${{ inputs.folder-to-exclude }}
```

Comment:
I'm wondering, does this work if we want to ignore several folders?

Reply:
yes it works on the validator with this
`folder-to-exclude: "! -path \"./vendor/*\" ! -path \"./files/modules/generator/*\""`